frb_rust/src/rust2dart.rs

```diff
@@ -139,3 +145,120 @@ impl<T: IntoDart> StreamSink<T> {
         self.rust2dart().close_stream()
     }
 }
+
+/// Basically the Into trait.
+/// We need this separate trait because we need to implement int for Vec<T> etc.
+pub trait IntoIntoDart<D> {
+    fn into_into_dart(self) -> D;
+}
+
+impl<T, D> IntoIntoDart<Vec<D>> for Vec<T>
+where
+    T: IntoIntoDart<D>,
+{
+    fn into_into_dart(mut self) -> Vec<D> {
+        self.drain(0..).map(|e| e.into_into_dart()).collect()
+    }
+}
+
+impl<A, AD, B, BD> IntoIntoDart<(AD, BD)> for (A, B)
```

Comment:
nit: given that we need to give a ton of implementations to IntoIntoDart, what about extract it to a separate file, say into_into_dart.rs

Reply:
Sounds good. I can also add tuple impls up to 5 or so in that file.
I maybe also missed some types if there are some that are not in the test but worked before.